pulp_deb/app/tasks/copy.py

```diff
@@ -0,0 +1,119 @@
+from django.db import transaction
+from django.db.models import Q
+
+from pulpcore.plugin.models import Content, RepositoryVersion
+
+from pulp_deb.app.models import AptRepository, Package, ReleaseArchitecture, PackageReleaseComponent
+
+import logging
+from gettext import gettext as _
+
+log = logging.getLogger(__name__)
+
+
+def find_structured_publish_attributes(content, src_repo_version):
+    """
+    Finds the content for structured publish from packages to be copied and returns it all together.
+
+    Args:
+        content (iterable): Content for structured publish
+        src_repo_version (pulpcore.models.RepositoryVersion): Source repo version
+
+    Returns: Queryset of Content objects that extends intial set of content for structured publish
+    """
+    # Content in the source repository version
+    package_release_component_ids = src_repo_version.content.filter(
+        pulp_type=PackageReleaseComponent.get_pulp_type()
+    ).only("pk")
+    architecture_ids = src_repo_version.content.filter(
+        pulp_type=ReleaseArchitecture.get_pulp_type()
+    ).only("pk")
+    package_release_components = PackageReleaseComponent.objects.filter(
+        pk__in=package_release_component_ids
+    )
+
+    children = set()
```

Comment:
```suggestion
    structured_publish_content = set()
```
If we can update all instances of `children` and `find_structured_publish_attributes` we are done!